### PR TITLE
doc/releases: USB DFU class upgrade record

### PR DESCRIPTION
**Pull request by nvlsianpu**
Added note on supported DFU for single-slot.

record describes what was added here: #28506

### CODE REVIEW

File: doc/releases/release-notes-2.5.rst [hunk 1]
```diff
@@ -173,6 +173,10 @@ Drivers and Sensors
 
 * USB
 
+  * Added support for operate only on the primary image by the DFU class which,
```
Comment (utzig) [inline doc/releases/release-notes-2.5.rst:176]: Do you really mean "primary image" or should it be "primary slot" (or "primary partition") ?

Comment (emob-nordic) [inline doc/releases/release-notes-2.5.rst:176]:
```suggestion
  * Support for USB DFU with single slot mode.
  * Support to use USB DFU within MCUBoot with single application slot mode.
```

**Review by eobalski — CHANGES_REQUESTED** [review]
I would reword this a bit.

File: doc/releases/release-notes-2.5.rst (hunk 1 above)
Comment (nvlsianpu) [reply doc/releases/release-notes-2.5.rst:176]: single slot mode is notion which belongs to MCUboot. Zephyr can provide single image partition.

Comment (nvlsianpu) [reply doc/releases/release-notes-2.5.rst]: @emob-nordic got better phrase from tech writer

**Review by eobalski — APPROVED** [review]
Way better.

**Review by carlescufi — APPROVED** [review]
